GET_STARTED.md

```diff
@@ -0,0 +1,442 @@
+# Get Started: Application Development using CCF
+
+## Overview
+
+Get started repository for building CCF applications using (JavaScript, TypeScript and C++).
+
+## Quick start
+
+What is CCF: The [Confidential Consortium Framework (CCF)](https://ccf.dev/) is an open-source framework for building a new category of secure, highly available,
+and performant applications that focus on multi-party compute and data.
+
+- Read the [CCF overview](https://microsoft.github.io/CCF/main/overview/index.html) and get familiar with [CCF's core concepts](https://microsoft.github.io/CCF/main/overview/what_is_ccf.html) and [Azure confidential computing](https://learn.microsoft.com/en-us/azure/confidential-computing/)
+- [Build new CCF applications](https://microsoft.github.io/CCF/main/build_apps/index.html) in TypeScript/JavaScript or C++
+- CCF [Modules API reference](https://microsoft.github.io/CCF/main/js/ccf-app/modules.html)
+- CCF application get started repos
+  - [CCF application template](https://github.com/microsoft/ccf-app-template)
+  - [CCF application samples](https://github.com/microsoft/ccf-app-samples)
+
+## Supported Programing Languages
+
+Applications can be written in
+
+- TypeScript
+- JavaScript
+- C++
+- More languages support upcoming on 2023
+
+## Development environment
+
+- Development container and VSCode [![Open in VSCode](https://img.shields.io/static/v1?label=Open+in&message=VSCode&logo=visualstudiocode&color=007ACC&logoColor=007ACC&labelColor=2C2C32)](https://vscode.dev/redirect?url=vscode://ms-vscode-remote.remote-containers/cloneInVolume?url=https://github.com/microsoft/ccf-app-samples)
```

Comment:
```suggestion
- VS Code Dev Container [![Open in VSCode](https://img.shields.io/static/v1?label=Open+in&message=VSCode&logo=visualstudiocode&color=007ACC&logoColor=007ACC&labelColor=2C2C32)](https://vscode.dev/redirect?url=vscode://ms-vscode-remote.remote-containers/cloneInVolume?url=https://github.com/microsoft/ccf-app-samples)
```

Reply:
Marketing get a bit funny with the name!

Reply:
Done